src/graphql/main/schema.graphql

```diff
@@ -1543,6 +1558,20 @@ type User {
   username: Username @deprecated(reason: "will be moved to @Handle in Account and Wallet")
 }
 
+type ReferralInfo {
+  referralLink: ReferralLink!
```

Comment:
Not the username?

Reply:
I think it can be just the username, but this gives us flexibility to set campaigns and custom links from the backend.